docs/ref/models/querysets.txt

```diff
@@ -2351,7 +2351,8 @@ exists in the database, an :exc:`~django.db.IntegrityError` is raised.
 
 This method inserts the provided list of objects into the database in an
 efficient manner (generally only 1 query, no matter how many objects there
-are), and returns created objects as a list, in the same order as provided:
+are), and returns created objects (including conflicted) as a list, in the
+same order as provided:
```

Comment:
hrm… we may be limiting the description even by saying this (also "created… including conflicted" itself sounds conflicting ;))

Reading the code, it returns all submitted instances _regardless_ if there are no errors. If the database supports `RETURNING` then those instances may be mutated with updated field values.

Reply:
That's right. I've updated the sentence, removing the confusing "created" part completely.